app/src/main/java/org/p2p/wallet/send/ui/NoTokensSendFragment.kt

```diff
@@ -0,0 +1,46 @@
+package org.p2p.wallet.send.ui
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import org.p2p.wallet.R
+import org.p2p.wallet.common.mvp.BaseFragment
+import org.p2p.wallet.databinding.FragmentSendNoTokenBinding
+import org.p2p.wallet.home.model.Token
+import org.p2p.wallet.moonpay.ui.new.NewBuyFragment
+import org.p2p.wallet.receive.solana.ReceiveSolanaFragment
+import org.p2p.wallet.utils.args
+import org.p2p.wallet.utils.popBackStack
+import org.p2p.wallet.utils.replaceFragment
+import org.p2p.wallet.utils.viewbinding.viewBinding
+import org.p2p.wallet.utils.withArgs
+
+private const val EXTRA_DEFAULT_TOKEN = "EXTRA_DEFAULT_TOKEN"
+
+class NoTokensSendFragment : BaseFragment(R.layout.fragment_send_no_token) {
```

Comment:
Maybe we should start naming with `Send`? 
For example: `SendDisabledFragment` or smth like that. 
The developer can have a context starting from the first word in the name

Reply:
im for SendNoTokensFragment